app/src/main/java/com/sopt/androidstudy/presentation/sign/viewmodels/LoginViewModel.kt

```diff
@@ -13,9 +13,10 @@ class LoginViewModel : ViewModel() {
     private val userEmail = MutableLiveData<String>()
     private val userPassword = MutableLiveData<String>()
     private val isEnabledButton = MediatorLiveData<Boolean>()
-    private val hi:String = "asdasd"
+    private val is_click = MutableLiveData<Boolean>(false)
```

Comment:
```suggestion
    private val isClicked = MutableLiveData<Boolean>(false)
```
모든 변수는 카멜케이스로 통일하시죠오!! 그리고 isClicked는 어떠신가요?

Reply:
여기도..... 파이썬의 저주인가 왜 스네이크 케이스 범벅이지..